module-system/module-implementations/sov-evm/src/query.rs

```diff
@@ -125,6 +125,27 @@ impl<C: sov_modules_api::Context> Evm<C> {
         Ok(Some(block.into()))
     }
 
+    #[rpc_method(name = "getTransactionCount")]
+    pub fn get_transaction_count(
+        &self,
+        address: reth_primitives::Address,
+        _block_number: Option<String>,
+        working_set: &mut WorkingSet<C>,
+    ) -> RpcResult<reth_primitives::U64> {
+        info!("evm module: eth_getTransactionCount");
+
+        // TODO: Implement block_number once we have archival state #882
```

Comment:
Let's remove `Fixes #882` from the description on this PR. This way, the issue remains open even after we merge this PR (until we have the archival node).